src/views/map.c

```diff
@@ -938,6 +1231,65 @@ static gboolean _view_map_motion_notify_callback(GtkWidget *widget, GdkEventMoti
   return FALSE;
 }
 
+static gboolean _view_map_scroll_event(GtkWidget *w, GdkEventScroll *event, dt_view_t *self)
+{
+  dt_map_t *lib = (dt_map_t *)self->data;
+  if(lib->loc.id > 0)
+  {
+    OsmGpsMapPoint *p = osm_gps_map_get_event_location(lib->map, (GdkEventButton *)event);
+    float lat, lon;
+    osm_gps_map_point_get_degrees(p, &lat, &lon);
+    if(dt_map_location_included(lon, lat, &lib->loc.data))
+    {
+      if(lib->loc.data.shape == MAP_LOCATION_SHAPE_RECTANGLE &&
+        (event->state & GDK_SHIFT_MASK))
+      {
+        if(event->direction == GDK_SCROLL_UP)
```

Comment:
this must be inverted for consistency with mask.

Reply:
OK